.github/workflows/run-phpunit.yml

```diff
@@ -0,0 +1,38 @@
+name: PHPUnit tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  phpstan:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['8.3']
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: mbstring, intl
+          coverage: yes
+
+      - name: php version
+        run: php -v
+
+      - name: Install Composer dependencies
+        run: composer install
```

Comment:
Nějaká cache by tady nastavit nešla? Nevím jak přesně fungují github actions, ale to s každým buildem budu instalovat všechny závislosti?

Reply:
not yet.